igvc_perception/include/pointcloud_filter/fast_segment_filter/fast_segment_filter.h

```diff
@@ -12,47 +12,77 @@
 
 #include <Eigen/Dense>
 
-typedef Eigen::Matrix<double, Eigen::Dynamic, 3> MatrixXd;
-typedef Eigen::Matrix<double, Eigen::Dynamic, 1> VectorXd;
+using MatrixXd = Eigen::Matrix<double, Eigen::Dynamic, 3>;
+using VectorXd = Eigen::Matrix<double, Eigen::Dynamic, 1>;
```

Comment:
Actually, now looking at it again doesn't Eigen already have typedefs for `MatrixXd` and `VectorXd`? instead of doing this you can do
```
using Eigen::MatrixXd;
using Eigen::VectorXd;
```